internal/tools/go.mod

```diff
@@ -14,7 +14,7 @@ require (
 	github.com/golangci/golangci-lint v1.53.3
 	github.com/google/addlicense v1.1.1
 	github.com/jstemmer/go-junit-report v1.0.0
-	github.com/open-telemetry/opentelemetry-collector-contrib/cmd/mdatagen v0.80.0
+	github.com/open-telemetry/opentelemetry-collector-contrib/cmd/mdatagen v0.81.0
```

Comment:
I think the resulting go.sum changes need to be included for the installation step to work

Reply:
sorry, fixed.